### DISCUSSION TITLE
F-Droid availability

### DISCUSSION THREAD

**Discussion by 5t0rmr1d3r** [ChilliBits/particulate-matter-app#8] [Ideas] [answered]
It would be great if you could make it available over F-Droid

**Comment by mrgames13** [comment, marked as answer]
We need to build our app without Google Play Services to upload it to F-Droid, but we didn't, as you correctly recognized in your issue #9. That means that we have to fix that issue first ;-)

**Comment by ghost** [comment]
+1 I would appreciate this as well. Also apps from F-Droid seem to be more trustworthy considering how much garbage and suspicious apps are available from Google Playstore.
